_plugins/markdown.rb

```diff
@@ -0,0 +1,23 @@
+=begin
```

Comment:
GitHub Pages will not build with any plugins, so you don't even have the option to use it